.github/workflows/publish_image_chart.yaml

```diff
@@ -54,6 +54,12 @@ jobs:
       - name: Build web static
         run: make build/web
 
+      # Setup QEMU and Buildx.      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@8b122486cedac8393e77aa9734c3528886e4a1a8 #v2.0.0
```

Comment:
I'm not sure which version should be use in this case so I put v2.0.0 for the time being @khanhtc1202

Reply:
buildx v2.0.0 is using `node16` 
 https://github.com/docker/setup-qemu-action/pull/48